src/components/output/OutputGesture.svelte

```diff
@@ -29,11 +30,8 @@
   }
   const wasTurnedOff = () => {};
   const wasTurnedOn = () => {
-    if (Microbits.isOutputMakecode()) {
-      ConnectionBehaviours.getOutputBehaviour().onGestureRecognized(
-        $gesture.ID,
-        $gesture.name,
-      );
+    if ($state.outputOrigin === HexOrigin.MAKECODE) {
+      sendToOutput['sendToOutputUart']('g', $gesture.name);
```

Comment:
I'm not sure we've made this clearer. Is this a stub?

Reply:
At the moment `sendToOutput` is an object that gets assigned functions when we have access to various 'BluetoothRemoteGATTCharacteristics' - see listenToOutputServices. It's the one chunk where Bluetooth still being a class would make sense, maybe.  Alternative approach is welcome.